Commands/summarize.py

```
@@ -1,119 +1,129 @@
-    except (ParseError, ExpatError) as e:
-        print(f"XML parsing error for video {video_id}: {e}")
-        return "API_ERROR"
     except Exception as e:
-        print(f"Unexpected error for video {video_id}: {e}")
-        return None
\ No newline at end of file
+        print(f"[Error] {e}")
+        return
+
+    finally:
+        return
```

Ruff (0.11.9)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)